chain/network/src/peer_manager.rs

```diff
@@ -163,13 +186,47 @@ pub struct PeerManagerActor {
     adv_disable_edge_pruning: bool,
 }
 
+impl AdvHelper {
+    #[cfg(not(feature = "test_features"))]
+    fn can_broadcast_edges(&self) -> bool {
+        true
+    }
+
+    #[cfg(feature = "test_features")]
+    fn can_broadcast_edges(&self) -> bool {
+        !self.adv_disable_edge_propagation
```

Comment:
for future: we should try to make sure that variable names (and meanings are matching)

here I'd rename it to self.adv_can_broadcast_edges

Reply:
Ok